ApleStore/ViewController/ForYouViewController.swift

```diff
@@ -7,10 +7,226 @@
 
 import UIKit
 
-final class ForYouViewController: UIViewController {
+enum Constraints {
+    static let viewLabelY = 190
+    static let viewLabelX = 240
+}
+
+final class ForYouViewController: UIViewController, UINavigationControllerDelegate {
```

Comment:
```suggestion
final class ForYouViewController: UIViewController {
```
, UINavigationControllerDelegate в ext